internal/collector/export_test.go

```diff
@@ -0,0 +1,26 @@
+package collector
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// Export for testing.
```

Comment:
It's not harmful but I really don't like this solution. :sweat_smile: 
I believe you either do white-box testing and let your test files be in the same package or you're going for black-box testing and separate your test files and in this case, it's a good thing to not have access to the private methods.

I also think it's okay to have both kinds of tests alongside each other.

And in this case, I think we should either go for exporting the methods in this package or just stick to the white-box tests that we have.
I recommend going with the latter one since it's easier to do and figure out later if it's a good idea or not to export the methods.
What do you think?

Reply:
Makes much more sense to me the same :D